### DISCUSSION TITLE
Failed subscription

### DISCUSSION THREAD

**Discussion by don-briski** [community/community#146890] [Copilot Conversations]
I had a monthly subscription done on my account twice(for December and January) which I was debited but the subscription failed. I have tried to get support but to no avail. Please how do I resolve this issue.

**Comment by Mohrip** [comment]
Hi,

To resolve the failed subscription issue, try the following steps:

Check Payment Method: Ensure your payment method is valid, not expired, and has enough funds. You can also try using a different payment method.

Verify Subscription Status: Check your subscription status in your GitHub account settings to confirm whether the payments were processed but the subscription wasn't activated.

Contact Support Again: If you haven't already, reach out to GitHub Support through their [contact form](https://support.github.com/contact). Provide details like transaction IDs and the dates of the failed payments.

Check for Issues on GitHub's End: Sometimes, GitHub experiences billing issues. Look for any announcements regarding service interruptions that may affect billing.

If the issue persists, ask GitHub Support to manually activate your subscription or refund the charges.
